### PR TITLE
#154999574 chore: refactor responses

### PR DESCRIPTION
**Pull request by Mercy-Muchai**
#### What does this PR do?
Adds more usable data on the GET responses
#### Description of Task to be completed?
- query parent tables
- return columns useful for the frontend
- rm filter endpoint by status
- rm unused imports in Notes controller
#### How should this be manually tested?
Use the routes file to test endpoint `GET` responses on postman
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[154999574](https://www.pivotaltracker.com/story/show/154999574)
#### Screenshots (if appropriate)
screen shot 2018-02-09 at 2 54 01 pm

### CODE REVIEW

**Comment by Mbarak-Mbigo** [comment]
Why don't we have the ids; `locationId, statusId, and leveId` in their `respective objects`?

**Comment by Mbarak-Mbigo** [comment]
Why r we using mixed conventions in naming keys? I believe we should stick to `camelCase`. It's good for consistency.

File: server/controllers/incidents.js [hunk 1]
```diff
@@ -10,44 +10,58 @@ module.exports = {
   create(req, res) {
     let locationPromise;
     name = req.body.name,
-    centre = req.body.centre,
-    country = req.body.country
+      centre = req.body.centre,
```
Comment (Mbarak-Mbigo) [inline server/controllers/incidents.js]: Remove unnecessary indentation. Align with `name`.

Comment (Mercy-Muchai) [reply server/controllers/incidents.js]: @Mbarak-Mbigo on the backend, we are using camel-case conventions for every column  but when referencing models in sequelize, we cannot use small case. We are reusing the models in different parts of the files. The nested objects however are in the expected conventions. Tried using aliases, doesn't work yet. cc: @kimobrian

**Pull request closed without merging**